dev/src/types.ts

```diff
@@ -321,6 +321,15 @@ export interface ValidationOptions {
   allowTransforms: boolean;
 }
 
+/**
+ * An options object that can be used to toggle the default rate limit
+ * throttling implemented by BulkWriter.
+ */
+export interface BulkWriterOptions {
```

Comment:
This needs to be added to the public d.ts file as well.

Reply:
added to firestore.d.ts, and also added option to pass in options to index.ts.